packages/terra-functional-testing/src/terra-cli/wdio/index.js

```diff
@@ -5,22 +5,91 @@ const cli = {
   describe: 'Run wdio tests',
   builder: (yargs) => (
     yargs.options({
+      assetServerPort: {
+        type: 'number',
+        describe: 'The port to run the webpack and express asset services on.',
+        default: () => {
+          if (process.env.WDIO_INTERNAL_PORT) {
+            return process.env.WDIO_INTERNAL_PORT;
+          }
+
+          return 8080;
+        },
+      },
       browsers: {
         type: 'array',
         describe: 'A list of browsers for the test run.',
+        default: () => {
+          let browsers = process.env.BROWSERS;
+          if (browsers) {
+            // Remove the brackets if BROWSERS is set as an array like ['chrome, firefox, ie'] or ['chrome','firefox','ie'] in Jenkinsfile.
+            browsers = browsers.replace('[', '');
+            browsers = browsers.replace(']', '');
```

Comment:
not a big deal, but we could reduce this to one replace call with a regex. if there's a chance there's more than one set of brackets, we could add the `/g` flag
```suggestion
            browsers = browsers.replace('/\[|\]/', '');
```

Reply:
Since we're parsing just the brackets, I thought the code is easier to read/understand this way. The regex is cool but the  escape and pipe characters makes it harder to read.